src/main/java/controllers/PostControllerPresenter.java

```diff
@@ -12,23 +10,28 @@
 import java.util.List;
 import java.util.UUID;
 
-import use_cases.RecipeManager;
-
-public class PostController {
+public class PostControllerPresenter {
```

Comment:
Are we not using the term "Controller" and not "ControllerPresenter"?

Reply:
I'm changing it to ControllerPresenter because it also displays posts (Presenter role)